intranet/static/js/common.nav.js

```diff
@@ -0,0 +1,39 @@
+$(function() {
```

Comment:
No, I (codacy) is letting you know that you're depending on jquery being included before this file is included.